includes/api/class-edd-api.php

```diff
@@ -101,6 +137,15 @@ class EDD_API {
 	 * @since 1.5
 	 */
 	public function __construct() {
+
+		$this->versions = array(
+			'v1' => 'EDD_API_V1',
```

Comment:
versions should be defined with just the number. The "v" part should only be for stylistic purposes added to the endpoint at the end. API endpoint versions should always only be defined by an integer number >=1 This prevents future issues and allows people to rename the api endpoints if they wanted to do that.